### ISSUE TITLE
ISO Hash Key problems

### ISSUE THREAD

**Issue by DavidK-M** [kiibohd/KiiConf#79]
Hi All,

I have configured a key as an ISO#, but it does not work. (The key is ignored...)

The appropriate snippet of the JSON looks like:

        {
            "code": "0x29",
            "x": 14,
            "y": 16,
            "w": 4,
            "h": 4,
            "layers": {
                "0": {
                    "key": "HASH",
                    "label": "ISO#"
                }
            }
        },

The serial output of the matrix debug gives:

WARNING - USB Code not within 4-49 (0x4-0x31), 51-155 (0x33-0x9B), 157-164 (0x9D-0xA4), 176-221 (0xB0-0xDD) or 224-231 (0xE0-0xE7) NKRO Mode: 0x32
WARNING - USB Code not within 4-49 (0x4-0x31), 51-155 (0x33-0x9B), 157-164 (0x9D-0xA4), 176-221 (0xB0-0xDD) or 224-231 (0xE0-0xE7) NKRO Mode: 0x32

Best regards,

David.

**Comment by DavidK-M** [comment]
This could be effectively the same as issue 193 in the controller:

See:
  https://github.com/kiibohd/controller/issues/193

Hopefully this isn't seen as cross-posting ;-)

**Comment by DarkDefender** [comment]
I think this is related to #78.

TL;DR: You can work around this problem by binding the key to ANSI backslash "\" instead. Your computer OS will correctly convert the keycode to the hash key if you are on a ISO keyboard layout
